docs/openapi/components/schemas/common/Entity.yml

```diff
@@ -20,6 +20,9 @@ properties:
     enum:
       - Person
       - Organization
+    $linkedData:
+      term: entityType
+      '@id': https://schema.org/additionalType
```

Comment:
fyi, this is why this does not fix types when you look at RDF view... because the entityType is not type, RDF does not pick up term definitions for it.